components/config/prefs.rs

```diff
@@ -150,9 +171,13 @@ pub struct Preferences {
     // https://testutils.spec.whatwg.org#availability
     pub dom_testutils_enabled: bool,
     /// Enable WebGL2 APIs.
```

Comment:
Nit: remove this line

Reply:
Removing it as part of this PR feels weird.